src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -236,7 +236,7 @@
      {"methodVersion" (int (common/get-text refs "snapshotId"))})
    :render
    (fn [{:keys [props refs state this]}]
-     [:div {} (if (:parse-error? props) (style/create-server-error-message "ERROR: This method does not pass WDL validation."))
+     [:div {} (if-let [wdl-parse-error (:wdl-parse-error props)] (style/create-server-error-message wdl-parse-error))
```

Comment:
An `if` with no "else" should be a `when`.  Also, another way to write the entire if/when would be:
```clojure
(some-> (:wdl-parse-error props) style/create-server-error-message)
;; or
(some-> props :wdl-parse-error style/create-server-error-message)
```
`some->` is like `->` but it stops if the value ever becomes `nil`.